custom_components/opensprinkler/__init__.py

```diff
@@ -412,36 +273,25 @@ def device_state_attributes(self):
         for attr in [
             "name",
             "index",
-            "enabled",
-            "is_running",
             "is_master",
             "running_program_id",
             "seconds_remaining",
-            "start_time",
-            "max_name_length",
-            "master_1_operation_enabled",
-            "master_2_operation_enabled",
             "rain_delay_ignored",
             "sensor_1_ignored",
             "sensor_2_ignored",
             "sequential_operation",
```

Comment:
Curious how `sequential_operation` attribute would be used ?

Reply:
Don't know happy to remove

Reply:
removed all the _ignored ones as well